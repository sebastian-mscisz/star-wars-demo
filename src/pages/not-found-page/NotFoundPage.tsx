import { ErrorComponent } from 'src/components';

export const NotFoundPage: React.FC = () => (
    <ErrorComponent code={'404'} message={'This is not the page you were looking for!'} />
);
