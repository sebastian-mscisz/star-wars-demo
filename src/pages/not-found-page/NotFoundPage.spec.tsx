import { BrowserRouter } from 'react-router-dom';
import { NotFoundPage } from './NotFoundPage';
import { render } from '@testing-library/react';

describe('Not Found Page', () => {
    it('should render', () => {
        const { asFragment } = render(<NotFoundPage />, { wrapper: BrowserRouter });
        expect(asFragment()).toMatchSnapshot();
    });
});
