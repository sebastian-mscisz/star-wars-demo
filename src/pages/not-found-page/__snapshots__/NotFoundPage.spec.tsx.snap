// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page should render 1`] = `
<DocumentFragment>
  <div
    class="sw-error"
  >
    <h1>
      404 This is not the page you were looking for!
    </h1>
    <button
      class="sw-error__button"
    >
      <a
        aria-current="page"
        class="active"
        href="/"
      >
        Go home
      </a>
    </button>
  </div>
</DocumentFragment>
`;
